molecule/mtls-custombundle-rhel/molecule.yml

```diff
@@ -113,9 +113,6 @@ platforms:
     networks:
       - name: confluent
 provisioner:
-  playbooks:
-    prepare: prepare.yml
```

Comment:
This appears to be a change, but Molecule might have that prepare-playbook path as the default?

Reply:
ya so if you provide a playbook named "prepare.yml" this line is not needed. that playbook will get used as for the prepare phase (which is what happens). Only if that playbook had a different name is this line necessary